paging/common/src/test/kotlin/androidx/paging/InvalidatingPagingSourceFactoryTest.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.paging
+
+import org.junit.Test
+import kotlin.test.assertEquals
+
+
+class InvalidatingPagingSourceFactoryTest {
+
+    @Test
+    fun getPagingSource() {
+
+        val testFactory = object : InvalidatingPagingSourceFactory<Int, Int>() {
+            override fun create(): PagingSource<Int, Int> = TestPagingSource();
+        }
```

Comment:
Instead of copying these lines into every test, pull it out into a class-level property or make a helper function / class